### ISSUE TITLE
No Time-Series Regional Data for the US 

### ISSUE THREAD

**Issue by therealcyberlord** [CSSEGISandData/COVID-19#1713]
Hi, 

Please add U.S. regional data for the time series folder. In addition, please switch back to using states rather than counties. States are so much cleaner than counties. 

Thank you for your work and effort.

**Comment by Anna1077** [comment]
Following.

**Comment by Tboveri** [comment]
Yes would like US time series data by state and/or county level would be greatly appreciated!

**Comment by rgoel17** [comment]
Yes please!!

**Comment by davidbau** [comment]
Dupe of #1505

**Comment by rks125** [comment]
You can get the John Hopkins data in the format you are requesting, here:

https://www.soothsawyer.com/john-hopkins-time-series-data-confirmed-case-csv-after-march-22-2020/?github=7

I have been maintaining for five days now.

**Comment by Tboveri** [comment]
Thanks very much!

_

Thierry Boveri, CGFM  Senior Manager
O  407 628 2600  /  M  407 421 1400  /  E  tboveri@raftelis.com<mailto:tboveri@raftelis.com>
prmginc.com<http://www.prmginc.com/>  /  raftelis.com<http://www.raftelis.com/>  /  PRMG is now part of Raftelis. Learn More<https://www.raftelis.com/raftelis-acquires-prmg/>



Raftelis is a Registered Municipal Advisor within the meaning as defined in Section 15B (e) of the Securities Exchange Act of 1934 and the rules and regulations promulgated thereunder (Municipal Advisor Rule). However, except in circumstances where Raftelis expressly agrees otherwise in writing, Raftelis is not acting as a Municipal Advisor, and the opinions or views contained herein are not intended to be, and do not constitute “advice” within the meaning of the Municipal Advisor Rule. This message may contain information that is confidential or privileged. If you are not the intended recipient, please advise the sender and immediately delete this message.
